istioctl/cmd/root.go

```diff
@@ -50,6 +50,9 @@ var (
 	// Create a kubernetes.ExecClient (or mockExecClient)
 	clientExecFactory = newExecClient
 
+	// Create a kubernetes.ExecClientSDS
+	clientExecFactorySds = newSDSExecClient
```

Comment:
Factory should be the last word in the var name. IMO

Reply:
fixed